### PR TITLE
feat: treesitter support for swift

### PR DESCRIPTION
**Pull request by simpleliujl**

### CODE REVIEW

File: tests/treesitter/swift_test.swift [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        }
+    }
+    init<T, V>(_ sss: T, xxx: V) {
+
+    }
+}
+
+// TODO: Some stuff
+// FIXME: Some fix
+// MARK: Some mark
```
Comment (stevearc) [inline tests/treesitter/swift_test.swift:79]: Any reason these are included?

**Review by stevearc — APPROVED** [review]
Generally LGTM, one question

File: tests/treesitter/swift_test.swift (hunk 1 above)
Comment (simpleliujl) [reply tests/treesitter/swift_test.swift:79]: These are not necessory, I will delete them.

**Comment by stevearc** [comment]
Thanks for the PR!

**Pull request merged**